components/zoom_admin/zoom_admin.app.mjs

```diff
@@ -0,0 +1,374 @@
+      optional: true,
+    },
+    webinar: {
+      type: "string",
+      label: "Webinar",
+      optional: true,
+      description: "The Webinar ID",
+      async options({
+        prevContext,
+        page,
+      }) {
+        if (!prevContext.nextPageToken && page > 0) {
+          return [];
+        }
+
+        const {
+          webinars,
+          next_page_token,
+        } = await this.listWebinars({
+          nextPageToken: prevContext.nextPageToken,
+          pageSize: 2,
+        });
+        if (!webinars.length) {
+          return [];
+        }
+        const rawOptions = webinars.map((w) => ({
+          label: w.topic,
+          value: {
+            label: w.topic,
+            value: w.id,
+          },
+        }));
+        const options = sortBy(rawOptions, [
+          "label",
+        ]);
+
+        return {
+          options,
+          context: {
+            nextPageToken: next_page_token,
+          },
+        };
+      },
+    },
+  },
+  methods: {
+    _apiUrl() {
+      return "https://api.zoom.us/v2";
+    },
+    _getHeaders() {
+      return {
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+        "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
+        "user-agent": "@PipedreamHQ/pipedream v0.1",
+      };
+    },
+    _getAxiosParams(opts = {}) {
+      return {
+        ...opts,
+        url: this._apiUrl() + opts.path,
+        headers: this._getHeaders(),
+      };
+    },
+    async _makeRequest(opts) {
+      if (!opts.headers) opts.headers = {};
+      opts.headers["Accept"] = "application/json";
+      opts.headers["Content-Type"] = "application/json";
+      opts.headers["Authorization"] = `Bearer ${this.$auth.oauth_access_token}`;
+      opts.headers["user-agent"] = "@PipedreamHQ/pipedream v0.1";
+      const { path } = opts;
+      delete opts.path;
+      // eslint-disable-next-line multiline-ternary
+      opts.url = `${this._apiUrl()}${path[0] === "/" ? "" : "/"}${path}`;
+      return await axios(opts);
+    },
+    sanitizedArray(value) {
+      if (isArray(value)) {
+        return value.map((item) => get(item, "value", item));
+      }
+
+      // If is string, try to convert it in an array
+      if (isString(value)) {
+        // Return an empty array if string is empty
+        if (isEmpty(value)) {
+          return [];
+        }
+
+        return value.replace(/["'[\]\s]+/g, "").split(",");
+      }
+
+      throw new Error(`${value} is not an array or an array-like`);
+    },
+    async listMeetings(nextPageToken) {
+      const { data } = await this._makeRequest({
+        path: "/users/me/meetings",
+        params: {
+          page_size: 100,
+          next_page_token: nextPageToken,
+        },
+      });
+
+      return data;
```

Comment:
If you use pipedream's axios you can change this code as follows
```javascript
      return this._makeRequest({
        path: "/users/me/meetings",
        params: {
          page_size: 100,
          next_page_token: nextPageToken,
        },
      });
```

Reply:
Yeah, I will rebuild the http handler

Reply:
@jcortes As far as I know. It is not possible to use pipedream Axios here because I haven't access to $ when I am filling the options. The $ variable is only accessible through the run function that happens after the options. If you have some solution. Please let me know

Reply:
@vellames-turing please see my comment [above](https://github.com/PipedreamHQ/pipedream/pull/1866#discussion_r758523787)